src/build/__main__.py

```diff
@@ -96,6 +97,8 @@ def main_parser():  # type: () -> argparse.ArgumentParser
     """
     cwd = os.getcwd()
     out = os.path.join(cwd, 'dist')
+    # https://github.com/python/mypy/issues/1422
```

Comment:
```suggestion
    # mypy does not recognize module.__path__
    # https://github.com/python/mypy/issues/1422
```

It'd be nice to add a comment explaining what this is. Saves people a click.